docs/development.md

```diff
@@ -398,6 +398,19 @@ Response: "{\"temperature_degrees_celsius\":29}"
 [07:56:11 ✓:1,✗:0,⠇:0] └[oak-functions examples]─▶[OK] 17s
 ```
 
+## Extracing vmlinux from your Linux installation
+
+On Linux installations, you can extract the uncompressed Linux kernel ELF binary
+from the compressed kernel at `/boot/vmlinuz-$(uname -r)`. You will need the
+[extract-vmlinux](https://elixir.bootlin.com/linux/v6.2.8/source/scripts/extract-vmlinux)
```

Comment:
optional: Perhaps use the latest script from the main linux repo (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/plain/scripts/extract-vmlinux) or the github mirror (https://raw.githubusercontent.com/torvalds/linux/master/scripts/extract-vmlinux), although it is probably the same and seems unlikely to change.

Reply:
Done.